docs/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Bug report
+about: Create an to report bug
+title: 'Bug: '
+labels: Bug, Triage
+assignees: ''
```

Comment:
Please specify here how we need to add assignees. Like @username or just username etc